docs/Developers/Newcomers_Guide.rst

```diff
@@ -343,9 +343,10 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
     However those who are familiar with opensource can start with
     ``difficulty/low`` issues.
 
-    We highly encourage you to start reviewing other's ``difficulty/low`` or
-    ``difficulty/newcomer`` issues after you complete your newcomer issue, as
-    reviewing helps you to learn more about coala and python.
+    We highly encourage you to start `reviewing <https://coala.io/review>`__
+    other's ``difficulty/low`` or ``difficulty/newcomer`` issues after you
```

Comment:
there is no such link that provide such type of sorted list....atleast this link lead them to pending PRs or they will end up around wondering for tht.....and they will by themself review only those PRs in which they will find them comfortable........or else if u say i will remove tht link @adtac